pyomo/core/expr/visitor.py

```diff
@@ -12,28 +12,17 @@
 
 import inspect
 import logging
-import six
 from copy import deepcopy
 from collections import deque
 
-if six.PY2:
-    getargspec = inspect.getargspec
-else:
-    # For our needs, getfullargspec is a drop-in replacement for
-    # getargspec (which was removed in Python 3.x)
-    getargspec = inspect.getfullargspec
+getargspec = inspect.getfullargspec
```

Comment:
We should either `from inspect import getfullargspec` or just use `inspect.getfullargspec` below.